scripts/app.js

```diff
@@ -300,10 +301,82 @@ function displayBlog(){
      })
      
  }
-//  <td>1</td>
-//                                     <td>React Dommination until 2021</td>
-//                                     <td>Nelly Diane</td>
-//                                     <td>12-08-2020</td>
-//                                     <td onclick="triggerReadCommentsModal('open')"><img src="../assetes/icons/comments-solid.svg" alt="comment-icon"></td>
-//                                     <td onclick="triggerEditBlogModal('open')"><img src="../assetes/icons/pen-solid.svg" alt="pen icon"></td>
-//                                     <td><img src="../assetes/icons/trash-alt-regular.svg" alt="trash icon"></td>
\ No newline at end of file
+
+ 
+let newBlogImage="";
+function getNewBlogImage(event){
+   if(event.target.files[0] != null){
+       newBlogImage = event.target.files[0];
+   }
+} 
+function updateBlog(){
+    let blogId = document.getElementById('blogId').value;
+    let blogHeader = document.getElementById('blogHeader').innerHTML;
+    let blogContent = document.getElementById('blogContent').innerHTML;
+    
+    let successToaster =  document.getElementById('successToaster')
+    let successMessage = document.getElementById('successMessage');
+
+    
+    successToaster.classList.toggle('active');
+    successMessage.innerHTML = "Updating blog.."
+
+    if(newBlogImage != ""){
+        storage.ref(`blogs/${blogId}/blogImage.jpg`).put(newBlogImage)
+    }
+    db.collection('blogs').doc(blogId).update({
+        title: blogHeader,
+        blogBody: blogContent
+    })
+    .then(() => {
+       successMessage.innerHTML = "Blog updted"
+        setTimeout(()=>{
+            successToaster.classList.remove('active')
+        },3000)
+        retreiveBlogs()
+    })
+    .catch(function(error) {
+        // The document probably doesn't exist.
+       alert("Error updating document: ", error);
```

Comment:
Get a better way of displaying user feedback than alerts

Reply:
Noted